dev/cnf/dependabot/check_this_in_if_it_changes/pom.xml

```diff
@@ -2539,12 +2539,12 @@
     <dependency>
       <groupId>org.bouncycastle</groupId>
       <artifactId>bcpkix-jdk18on</artifactId>
-      <version>1.75</version>
+      <version>1.78.1</version>
```

Comment:
You should run `./gradlew cnf:dependabotGenerate` to update this file and put the file in.  In reality these entries go away since they are already there after this.